packages/manager/modules/pci/src/projects/project/sidebar/project-list/project-list.html

```diff
@@ -50,13 +50,19 @@
                     data-ng-repeat="project in $ctrl.projects | filter:$ctrl.searchText track by $index"
                 >
                     <a
-                        ui-sref="pci.projects.project({ projectId: project.project_id })"
+                        data-ng-click="$ctrl.goToProjectDetails(project)"
                         class="d-flex text-white"
                     >
                         <span
                             class="text-truncate"
                             data-ng-bind="project.description"
                         ></span>
+                        <span
+                            data-ng-if="project.isSuspended() || project.hasPendingDebt()"
+                            class="oui-icon project-warning-icon oui-icon-warning-circle"
```

Comment:
```suggestion
                            class="oui-icon oui-icon-warning-circle"
```
custom class still needed ?

Reply:
This is required to position the icon properly and also to change the icon color